br/pkg/lightning/mydump/region.go

```diff
@@ -34,6 +34,8 @@ const (
 	tableRegionSizeWarningThreshold int64 = 1024 * 1024 * 1024
 	// the increment ratio of large CSV file size threshold by `region-split-size`
 	largeCSVLowerThresholdRation = 10
+	// TableFileSizeINF for compressed size, for lightning 10TB is a relatively big value and will strongly affect efficiency
+	TableFileSizeINF = 10 * 1024 * tableRegionSizeWarningThreshold
```

Comment:
Sounds like a threshold for lightning. Maybe `MaxTableFileSize`?

Reply:
```
set fileSize to INF to make sure compressed files can be read until EOF. Because we can't get the exact size of the compressed files.
```
I would expect to put its usage in the comment (as well as why this works, maybe)

Reply:
addressed in https://github.com/pingcap/tidb/pull/39153/commits/46ddf535f8ebbea9c97ec56c16bec6146e701389